recipes-images/packagegroups/packagegroup-provision-tpm.bb

```diff
@@ -0,0 +1,10 @@
+DESCRIPTION = "Packagegroup for TPM2 provisioning"
+LICENSE = "MIT"
+
+inherit packagegroup
+
+SUMMARY_packagegroup-provision-tpm = "TPM 2.0 provisioning"
```

Comment:
I think this can also be cleaned up? Just remove the package specific from `SUMMARY`.